### DISCUSSION TITLE
Are alerts heavily delayed?

### DISCUSSION THREAD

**Discussion by cbortz** [pwillworth/galaxyharvester#24] [Q&A]
On the server Empire in Flames (ID 64), I have some alerts for OQ on minerals and chemicals. About 6 days ago, there was a 1000 OQ mineral added that I just happened to see. Shortly after I checked for the current best uses and it all populated as usual.

However, I never received an alert... at least not until today.

> ![image](https://user-images.githubusercontent.com/184307/119917477-a5745000-bf34-11eb-840a-05f2e7c7ba1d.png)
> spawn 6 days ago

> ![image](https://user-images.githubusercontent.com/184307/119917523-b8872000-bf34-11eb-82b6-57a907bdc92b.png)
> alerts for it 1 hour ago

I don't recall there being this long of a delay for notifications. Is the queue backed up?

**Comment by pwillworth** [comment]
Yeah, looks like something bogged it down.  Investigating and working on getting it caught up.  4 days behind at this time.
